platform/qt/include/qmapboxgl.hpp

```diff
@@ -174,6 +174,8 @@ class Q_DECL_EXPORT QMapboxGL : public QObject
 
     void removeAnnotation(QMapbox::AnnotationID);
 
+    void setLayoutProperty(const QString& layer, const QString& property, const QVariant& value);
```

Comment:
Let's use the Qt style on the header files e.g. put the `&` together with the param name.